docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,203 @@
+
+## Reading guide
+
+If you are reading this topic with the questions listed below, click them to jump to their answers.
+
+* [What do I need to do before deploying Nebula Graph?](#prerequisites)
+* [How to fast deploy Nebula Graph with Docker Compose?](#how-to-deploy)
+* [How to check the status and ports of the Nebula Graph service?](#check-the-nebula-graph-service-status-and-ports)
+* [How to check the data and logs of the Nebula Graph service?](#check-the-service-data-and-logs)
+* [How to stop the Nebula Graph Services?](#stop-the-nebula-graph-services)
+* [What are the other ways to install Nebula Graph?](#other-ways-to-install-nebula-graph)
+
+## Prerequisites
+
+* You have installed [Docker](https://docs.docker.com/engine/install/), [Docker Compose](https://docs.docker.com/compose/install/), and [Git](https://git-scm.com/download/linux) on your host.
+
+    >**NOTE**:
+    >
+    >* To use Docker as a non-root user, follow the steps in [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+    >* We recommend that you install the latest version of these applications to make sure they work properly.
+
+* You have started the Docker service on your host.
+
+* If you have already deployed another version of Nebula Graph with Docker Compose on your host, to avoid compatibility issues，back up [the service data](#check-the-service-data-and-logs) if you need, and delete the `nebula-docker-compose/data` directory.
+
+## How to deploy
+
+1. Clone the `nebula-docker-compose` repository to your host with Git.
+
+    * To install Nebula Graph 1.0, clone the `master` branch.
+
+    ```shell
+    $ git clone https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
+    * To install Nebula Graph 2.0-pre, clone the `v2-preview` branch.
+
+    ```shell
+    $ git clone --branch v2-preview https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
+2. Go to the `nebula-docker-compose` directory.
+
+    ```shell
+    $ cd nebula-docker-compose/
+    ```
+
+3. Run the following command to start all the Nebula Graph services.
+
+    ```shell
+    $ docker-compose up -d
+    ```
+
+    The following information indicates the services have started:
+
+    ```shell
+    Creating nebula-docker-compose_metad2_1 ... done
+    Creating nebula-docker-compose_metad1_1 ... done
+    Creating nebula-docker-compose_metad0_1 ... done
+    Creating nebula-docker-compose_storaged2_1 ... done
+    Creating nebula-docker-compose_graphd_1    ... done
+    Creating nebula-docker-compose_storaged0_1 ... done
+    Creating nebula-docker-compose_storaged1_1 ... done
+    ```
+
+    >**NOTE**: For more information of the preceding services, see [Design and Architecture of Nebula Graph](https://docs.nebula-graph.io/manual-EN/1.overview/3.design-and-architecture/1.design-and-architecture/).
+
+4. Use nebula-console to connect to Nebula Graph.
```

Comment:
this step could provide more information and it would be better if it comes later after the status check. nebula-console is an option. the other one is Studio. you could tell newcomers they have options. Nebula Graph is the center, so its documentation should provide more information to include its "derivatives" to help newcomers understand the ecosystem of Nebula Graph.

Reply:
The info you asked for is already provided below the step.